mathesar_ui/src/pages/schema/SchemaOverview.svelte

```diff
@@ -60,6 +88,27 @@
       <OverviewHeader title="Saved Explorations" />
       {#if isExplorationsLoading}
         <ExplorationSkeleton />
+      {:else if explorationsRequestStatus.state == 'failure'}
+        <ErrorBox>
+          <p>{explorationsRequestStatus.errors[0]}</p>
+          <div>
+            <Button
+              on:click={() => {
+                if ($currentSchemaId) {
+                  void refetchQueriesForSchema($currentSchemaId);
+                }
```

Comment:
Same comments as above. Utilize `SpinnerButton` instead of `Button` and the `schema` prop instead of the `currentSchemaId` store.